.github/actions/start-opensearch/action.yml

```diff
@@ -0,0 +1,127 @@
+name: 'Start OpenSearch'
```

Comment:
Thoughts on naming this: `Start OpenSearch with security`?

Reply:
The current idea is for this to really be usable with __any__ plugin. That being said it is currently only tested on security so that should be fine.